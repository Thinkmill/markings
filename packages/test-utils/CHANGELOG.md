# @markings/test-utils

## 0.0.1
### Patch Changes



- [`a417548`](https://github.com/Thinkmill/markings/commit/a4175484f2af47e9db1f17677e6d8a33066267e7) Thanks [@mitchellhamilton](https://github.com/mitchellhamilton)! - Initial Release
