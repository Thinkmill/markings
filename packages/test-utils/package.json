{
  "name": "@markings/test-utils",
  "version": "0.0.1",
  "main": "dist/test-utils.cjs.js",
  "module": "dist/test-utils.esm.js",
  "license": "MIT",
  "dependencies": {
    "@babel/core": "^7.9.6",
    "@babel/parser": "^7.8.6",
    "@babel/runtime": "^7.8.7",
    "@babel/traverse": "^7.8.6",
    "@manypkg/get-packages": "^1.0.0",
    "@markings/types": "^0.0.6",
    "@types/babel__core": "^7.1.6",
    "@types/babel__traverse": "^7.0.9",
    "@types/jest-in-case": "^1.0.1",
    "@types/parse-github-url": "^1.0.0",
    "chalk": "^2.4.2",
    "fs-extra": "^8.1.0",
    "globby": "^11.0.0",
    "jest-in-case": "^1.0.2",
    "meow": "^5.0.0",
    "normalize-path": "^3.0.0",
    "p-limit": "^2.2.1",
    "parse-github-url": "^1.0.2"
  },
  "devDependencies": {
    "fixturez": "^1.1.0",
    "strip-ansi": "^6.0.0"
  },
  "repository": "https://github.com/Thinkmill/markings/tree/master/packages/test-utils"
}
