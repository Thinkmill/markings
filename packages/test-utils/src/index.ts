export { snapshotMarkingsFromFixtures } from "./run-fixtures";
