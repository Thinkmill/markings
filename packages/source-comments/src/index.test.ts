import { snapshotMarkingsFromFixtures } from "@markings/test-utils";
import { source } from ".";

snapshotMarkingsFromFixtures(__dirname, source);
