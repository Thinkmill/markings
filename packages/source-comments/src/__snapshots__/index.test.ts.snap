// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fixme 1`] = `
// FIXME: something

      ↓ ↓ ↓ ↓ ↓ ↓

[
  {
    "description": "something",
    "purpose": "todo",
    "location": {
      "line": 1
    }
  }
]
`;

exports[`question 1`] = `
// QUESTION: a question

      ↓ ↓ ↓ ↓ ↓ ↓

[
  {
    "description": "a question",
    "purpose": "question",
    "location": {
      "line": 1
    }
  }
]
`;

exports[`todo 1`] = `
// TODO: thing

      ↓ ↓ ↓ ↓ ↓ ↓

[
  {
    "description": "thing",
    "purpose": "todo",
    "location": {
      "line": 1
    }
  }
]
`;
